rules/TypeDeclaration/Rector/FunctionLike/AddClosureParamTypeFromArgClassStringRector.php

```diff
@@ -125,37 +123,18 @@ private function processFunctionLike(
             return;
         }
 
-        if (is_int($addParamTypeForFunctionLikeWithinCallLikeArgFromArgDeclaration->getCallLikePosition())) {
-            if ($callLike->getArgs() === []) {
-                return;
-            }
-
-            $arg = $callLike->args[$addParamTypeForFunctionLikeWithinCallLikeArgFromArgDeclaration->getCallLikePosition()] ?? null;
-
-            if (! $arg instanceof Arg) {
-                return;
-            }
-
-            // int positions shouldn't have names
-            if ($arg->name !== null) {
-                return;
-            }
-        } else {
-            $args = array_filter($callLike->getArgs(), static function (Arg $arg) use (
-                $addParamTypeForFunctionLikeWithinCallLikeArgFromArgDeclaration
-            ): bool {
-                if ($arg->name === null) {
-                    return false;
-                }
-
-                return $arg->name->name === $addParamTypeForFunctionLikeWithinCallLikeArgFromArgDeclaration->getCallLikePosition();
-            });
```

Comment:
I've removed the `string` named args feature, as those are skipped now anyway. I'd limit this rule to work only with standard args in future as well, as area of use case is pretty narrow.